sys/behave/src/Behave.hs

```diff
@@ -72,26 +72,26 @@ import qualified Eval as Eval
 
 behInit :: [ Set.Set TxsDefs.ChanId ] -> TxsDefs.BExpr -> IOB.IOB (Maybe BTree)
 behInit chsets bexp  =  do
-     btree <- unfold chsets (BNbexpr Map.empty bexp)
-     return $ Just btree
+     btree' <- unfold chsets (BNbexpr Map.empty bexp)
+     return $ Just btree'
 
 
 -- ----------------------------------------------------------------------------------------- --
 -- behMayMenu :  may menu of BTree without quiescence
 
 
 behMayMenu :: [ Set.Set TxsDefs.ChanId ] -> BTree -> Menu
-behMayMenu chsets btree 
-  =  [ ( btoffs, hidvars, preds ) | BTpref btoffs hidvars preds next <- btree ]
-     ++ concat [ behMayMenu chsets btree' | BTtau btree' <- btree ]
+behMayMenu chsets btree' 
```

Comment:
Maybe in the future we need to avoid the use of `'`: it is very easy to make mistakes and pass the wrong variable around.

Reply:
I would like to have default names that don't shadow other operators.
E.g. btree is an operator therefore we should use bt for a BTree instance.